app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
+    pass
+
+
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
```

Comment:
Might will better to define a constructor for each class that set the message (pass it to base exception) and then in code just rise an exception without the message